.github/workflows/cron-tests.yaml

```diff
@@ -5,6 +5,7 @@ on:
     - cron: '0 20 * * *'
   push:
     branches:
+      - *
```

Comment:
Not sure if this is leftover from testing but i dont think this should be here

Reply:
There were an extra commit with some experiments, I just hadn't pushed the updated branch. Fixed.